simple_app/plots.py

```diff
@@ -48,18 +85,37 @@ def specplot(query: str, db_file: str, nightskytheme: Theme) -> Tuple[Optional[s
     failstr: str
         the failed spectra
     """
-    def normalise(fluxarr: np.ndarray) -> np.ndarray:
-        fluxarr = (fluxarr - np.nanmin(fluxarr)) /\
-                  (np.nanmax(fluxarr) - np.nanmin(fluxarr))
-        return fluxarr
+    def normalise() -> np.ndarray:
```

Comment:
Isn't it bad practice to have a function with no inputs? It operates on wave and flux, but it's not clear where they come from so depending on when `normalise` is being called you could get funny results.

Reply:
Oh, I realize now this is a function inside another function so you may have less issues with global variables (or maybe more, I never know how it goes)

Reply:
Added a docstring to explain it is grabbing 'wave' and 'flux' from the surrounding scope.